packages/@aws-cdk/aws-rds/test/cluster-engine.test.ts

```diff
@@ -59,7 +59,7 @@ describe('cluster engine', () => {
   test('cluster parameter group correctly determined for AURORA_MYSQL and given version', () => {
     // GIVEN
     const engine = DatabaseClusterEngine.auroraMysql({
-      version: AuroraMysqlEngineVersion.VER_2_07_1,
+      version: AuroraMysqlEngineVersion.VER_2_10_1,
```

Comment:
Let's leave the test as-is 🙂.

Reply:
Is there a reason why I can not change the test for this one?

Reply:
We generally don't touch existing tests unless there's a pressing need to change them for some reason, and I don't see anything like that here 🙂.